src/pages/playground.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
-import Banner from '../components/banner/banner';
+import Footer from '../components/footer/footer';
 
 import HeadFooter from '../components/headfooter/headfooter';
 export default class PlayGround extends React.Component {
 	render() {
 		return (
 			<HeadFooter>
-				<Banner/>
+				<Footer/>
```

Comment:
So we can put this `Footer` component in the `HeadFooter` component now! That's why I named it `HeadFooter`!